docker/localstack/localstack-init.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -e
```

Comment:
Sometimes localstack will fail to create things if they already exist, which breaks things that get setup after, which is very annoying. So I'm removing this to get the whole script to execute

Reply:
I didn't realise it was this opt that was getting in the way - no more `down` then `up` 🙌